cmake/RippledInterface.cmake

```diff
@@ -22,6 +25,8 @@ target_compile_definitions (opts
 target_compile_options (opts
   INTERFACE
     $<$<AND:$<BOOL:${is_gcc}>,$<COMPILE_LANGUAGE:CXX>>:-Wsuggest-override>
+    $<$<BOOL:${is_gcc}>:-Wno-maybe-uninitialized>
+    $<$<BOOL:${voidstar}>:-DENABLE_VOIDSTAR>
```

Comment:
how about moving this to target_compile_definitions ?

Reply:
> how about moving `ENABLE_VOIDSTAR` to target_compile_definitions ?

Done!